tests/ingresstests/crd_test.go

```diff
@@ -981,6 +981,80 @@ func TestHTTPRuleCreateDelete(t *testing.T) {
 	TearDownIngressForCacheSyncCheck(t, modelName)
 }
 
+func TestHTTPRuleCreateDeleteWithPkiRef(t *testing.T) {
```

Comment:
should we have test case of ssl ref field added as part of hostrule?

Reply:
sslref is separately taken care of in a different testcase. I believe there is no benefit in doing the same with pki profile too.